packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```diff
@@ -477,6 +478,7 @@ popd
 %{_cross_libdir}/nvidia/tesla/libnvidia-fbc.so.1
 %{_cross_libdir}/nvidia/tesla/libnvoptix.so.%{tesla_ver}
 %{_cross_libdir}/nvidia/tesla/libnvoptix.so.1
+%{_cross_datadir}/nvidia/nvoptix.bin
```

Comment:
Technically not a lib, but part of the `Graphics` mounts:

https://github.com/NVIDIA/nvidia-container-toolkit/blob/adb5e6719de18d0e97bd825693d657d121813166/internal/discover/graphics.go#L52

So its OK to keep it in this section as we originally grouped things depending on the mount group they belonged to.